webclient/components/papers/note-change/templates/add-form.vue

```diff
@@ -200,7 +200,7 @@
                 to use el-input. Hence, I am using simple input box for vue-tribute.
                 I am assigning a class 'el-input__inner' for same design as el-input.
                 -->
-              <vue-tribute :options="_fieldDef.tributeOptions">
+              <vue-tribute :options="doTributeOptions">
```

Comment:
When I have to choose the dx for the patient. I cannot give a medical term. Hence each entity should be able to decide the children of mergedTributeKeyValuePairs